index.bs

```diff
@@ -5425,8 +5425,64 @@ Process Evolution</h2>
 	the [[COC|Code of Conduct]] in the <a href="https://www.w3.org/community/pwe/">Positive Work Environment Community Group</a>,
 	and the Process in the <a href="https://www.w3.org/community/w3process/">W3C Process Community Group</a>.
 
-<h2 id="acks" class=non-normative>
-Acknowledgments</h2>
+<h2 id="old-terms" class="non-normative nonum">
+Appendix A: Retired Terminology</h2>
+
+	<em>This section is non-normative.</em>
+
+	Previous version of this process defined and used various concepts and document statuses
+	that have since been removed or renamed.
+	For ease of reference,
+	this appendix lists the more notable ones,
+	and it links to the relevant section
+	of most recent version of the Process that still used them,
+	or gives the more recent equivalent term when there is one.
+
+	Retired concepts:
```

Comment:
What is the ordering of these terms? Is there a reason it's not simply alphabetical by "old term"? (And can we be consistent about calling these "terms" (preferred) vs. "concepts" vs. "document statuses"?)

Does it make sense to divide the list into separate "retired" and "renamed" sections? 

Presumably, if I'm looking here, I don't know whether the term I'm looking for was retired or renamed, nor to what. These separate sections mean I have to look through both lists, and with the mysterious sort-order, I must *read* every term, rather than skimming to the appropriate alphabetic position.

I suggest that these sub-lists be blended, and the presentation modified to a table (rather than a definition list) with columns for —
1. old term
2. last use (linked to last definition)
3. newer term (if any) (linked to current/latest definition)

I suggest that these rows be alpha-ordered by "old term".

I suggest that the "last used" entries never mention (but always link to) the specific location of the term definition in the old document (rather than mentioning the specific section only for retired concepts).

I also suggest that the "newer term" entry always link to its definition in the *current* document.

Reply:
Attempted to address all this with editorial changes in https://github.com/w3c/process/pull/898/commits/59b609cb34863b595d063e7a27fa1dd26e81c5e0. Closing, but feel free to file new comments if your have follow up issues.